src/bundle/Resources/config/services/normalizers.yaml

```
@@ -0,0 +1,24 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    Ibexa\Search\Serializer\Normalizer\Suggestion\ContentSuggestionNormalizer:
+        tags:
+            - { name: 'ibexa.search.serializer.normalizer' }
+
+    Ibexa\Search\Serializer\Normalizer\Suggestion\ParentLocationCollectionNormalizer:
+        tags:
+              - { name: 'ibexa.search.serializer.normalizer' }
+
+    Ibexa\Search\Serializer\Normalizer\Suggestion\ParentLocationNormalizer:
+        tags:
+              - { name: 'ibexa.search.serializer.normalizer' }
```

Warning: this registers normalizers in the default app's serializer due to autoconfiguration.

```suggestion
    Ibexa\Search\Serializer\Normalizer\Suggestion\ContentSuggestionNormalizer:
        autoconfigure: false
        tags:
            - { name: 'ibexa.search.serializer.normalizer' }

    Ibexa\Search\Serializer\Normalizer\Suggestion\ParentLocationCollectionNormalizer:
        autoconfigure: false
        tags:
              - { name: 'ibexa.search.serializer.normalizer' }

    Ibexa\Search\Serializer\Normalizer\Suggestion\ParentLocationNormalizer:
        autoconfigure: false
        tags:
              - { name: 'ibexa.search.serializer.normalizer' }
```